Kickstarter-iOS/Features/BlockingUsers/BlockUserActionSheetView.swift

```diff
@@ -0,0 +1,25 @@
+import SwiftUI
+
+@available(iOS 15.0, *)
+struct BlockUserActionSheetView: View {
+  var blockUser: () -> Void
+  var viewProfile: (() -> Void)?
+
+  var body: some View {
+    // Scott TODO: Use localized strings once translations are done
+    EmptyView()
+      .confirmationDialog("", isPresented: .constant(true), titleVisibility: .hidden) {
+        if let viewProfile = viewProfile {
+          Button("View profile") {
+            viewProfile()
+          }
+        }
+
+        Button(role: .destructive) {
```

Comment:
nit: We should update the styling to be our red color and also a kickstarter font. But that can happen in a separate PR.